releasenotes/notes/refactor-docxtodocument-97cfc711445e77cd.yaml

```diff
@@ -0,0 +1,4 @@
+---
+enhancements:
+  - |
+    Nest the docx metadata under the metadata key 'docx' and save the metadata in a dataclass object called DocxMetadata.
```

Comment:
I'd remove the release note if we haven't released the component in a public build yet (can't tell from just this PR).

Reply:
Do I just leave the release note empty then? Or can we skip the reno check and delete the file?

Reply:
We can skip the file - i've added the ignore label to the PR.